src/main/kotlin/no/nav/syfo/client/azuread/AzureAdClient.kt

```diff
@@ -1,20 +1,21 @@
 package no.nav.syfo.client.azuread
 
 import io.ktor.client.call.*
+import io.ktor.client.engine.*
 import io.ktor.client.plugins.*
 import io.ktor.client.request.*
 import io.ktor.client.request.forms.*
 import io.ktor.client.statement.*
 import io.ktor.http.*
-import no.nav.syfo.client.httpClientProxy
+import no.nav.syfo.client.*
 import org.slf4j.LoggerFactory
 import java.util.concurrent.ConcurrentHashMap
 
 class AzureAdClient(
     private val azureEnvironment: AzureEnvironment,
+    private val engine: HttpClientEngine? = null,
```

Comment:
Kunne kanskje gjort noe ala
`private val httpClient: HttpClient = httpClientProxy()` i konstruktøren her (og i GraphApiClient) og droppet httpClient-variabelen under. Så heller sendt inn en egen httpClient med mock engine der de instansieres opp i testkoden. 
Så slipper vi også å legge til kode i HttpClientCommon bare for testene.